pkg/securitypolicy/securitypolicyenforcer_rego.go

```diff
@@ -251,6 +258,15 @@ func (policy *regoEnforcer) queryEnforcementPoint(enforcementPoint string) (*enf
 		return nil, fmt.Errorf("enforcement point rule %s is invalid", enforcementPoint)
 	}
 
+	versionMissing, err := result.Bool("version_missing")
+	if err != nil {
+		return nil, err
+	}
+
+	if versionMissing {
+		return nil, errors.New(noAPIVersionError)
```

Comment:
Rather than using `errors.New` here, if you make the error types exported and define the global variables with `errors.New` above, it would be possible in the future to use `errors.Is` to take different actions based on different issues and decouple from the exact wording in the error message.